{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2020-35381"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "jsonparser 1.0.0 allows attackers to cause a denial of service (panic: runtime error: slice bounds out of range) via a GET call."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://github.com/buger/jsonparser/issues/219",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/buger/jsonparser/issues/219"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2020-12-15T21:26Z",
  "publishedDate": "2020-12-15T21:15Z"
}